modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/search_bar_configuration_suggestions/contributor_input_set_item/inputs/CharacterThresholdInput.js

```diff
@@ -34,13 +40,14 @@ function CharacterThresholdInput({onBlur, onChange, touched, value}) {
 
 			<ClayInput
 				aria-label={Liferay.Language.get('character-threshold')}
+				id={`character-threshold-${index}`}
```

Comment:
IDs are normally camelCased so this should be:

```
id={`characterThreshold${index}`}
```

Same for all the other `id` and `htmlFor` changes in this PR.